content/sensu-go/6.0/sensuctl/back-up-recover.md

```diff
@@ -129,24 +129,55 @@ When you export users, required password attributes are not included.
 You must add a [`password_hash`](../#generate-a-password-hash) or `password` to `users` resources before restoring them with the `sensuctl create` command.
 {{% /notice %}}
 
-## List types of supported resources
+## Sensuctl dump resource types
```

Comment:
I don't think you need to change this because these resource types are supported in other commands, ex. `sensuctl prune`.

Reply:
```suggestion
## Supported resource types
```